pkg/api/entries.go

```diff
@@ -183,7 +183,14 @@ func createLogEntry(params entries.CreateLogEntryParams) (models.LogEntry, middl
 	metricNewEntries.Inc()
 
 	queuedLeaf := resp.getAddResult.QueuedLeaf.Leaf
-	uuid := hex.EncodeToString(queuedLeaf.GetMerkleLeafHash())
+
+	hexEncodedHash := hex.EncodeToString(queuedLeaf.GetMerkleLeafHash())
+	entryID, err := sharding.CreateEntryIDWithActiveTreeID(hexEncodedHash)
+	if err != nil {
+		err := fmt.Errorf("error creating EntryID from active treeID and uuid %v: %v", hexEncodedHash, err)
```

Comment:
nit if you end up re-spinning this:
```suggestion
		err := fmt.Errorf("error creating EntryID from active treeID and uuid %v: %w", hexEncodedHash, err)
```